content/master/concepts/sever-side-apply.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Server-Side Apply
+state: alpha
+alphaVersion: "1.15"
```

Comment:
It would be ideal to clarify that only claim SSA (the first heading) is alpha.

That said, I think this is fine as-is. Better to be a bit scary and frame all of SSA as alpha, even if the functions part of it is beta, than to risk it seeming more mature than it is.